react-components/src/tools/details/ToolInfoDialog.js

```diff
@@ -0,0 +1,227 @@
+        setTabIndex(value);
+    };
+    const classes = useStyles();
+
+    let labelClass = classes.detailsLabel,
+        valueClass = classes.detailsValue;
+
+    return (
+        <React.Fragment>
+            <Dialog open={dialogOpen} maxWidth="sm" id={baseDebugId}>
+                <DEDialogHeader
+                    heading={selectedTool ? selectedTool.name : ""}
+                    onClose={onClose}
+                    id={baseDebugId}
+                />
+                <DialogContent style={{ minHeight: 400 }}>
+                    <LoadingMask loading={loading}>
+                        <Tabs
+                            indicatorColor="primary"
+                            textColor="primary"
+                            value={tabIndex}
+                            onChange={handleTabChange}
+                            variant="scrollable"
+                            scrollButtons="auto"
+                        >
+                            <Tab
+                                label={formatMessage(
+                                    intl,
+                                    "toolInformationLbl"
+                                )}
+                            />
+                            <Tab
+                                label={formatMessage(intl, "appsUsingToolLbl")}
+                            />
+                        </Tabs>
+                        {tabIndex === 0 && tool && (
+                            <Paper id={baseDebugId} style={{ padding: 5 }}>
+                                <Grid
+                                    container
+                                    spacing={2}
+                                    style={{ paddingLeft: 5 }}
+                                >
+                                    <Grid item xs={12}>
+                                        <span className={labelClass}>
+                                            {getMessage("toolAttributionLabel")}
+                                            :
+                                        </span>
+                                        <span className={valueClass}>
+                                            {selectedTool.attribution}
+                                        </span>
+                                    </Grid>
+                                    <Grid item xs={12}>
+                                        <span className={labelClass}>
+                                            {getMessage("descriptionLabel")}:
+                                        </span>
+                                        <span className={valueClass}>
+                                            {selectedTool.description}
+                                        </span>
+                                    </Grid>
+                                    <Grid item xs={12}>
+                                        <Typography variant="h6">
+                                            {getMessage("restrictionsLabel")}
```

Comment:
I know the original tool info dialog only included the attribution, description, and a few restrictions, but while we're making a new info dialog, I'd like to add more info to it.

It would be nice to be able to see more of the tool's container restrictions:
```javascript
tool.container.max_cpu_cores
tool.container.min_cpu_cores
tool.container.min_memory_limit
tool.container.min_disk_space
```
These could be helpful to an app integrator that needs to set some default/hidden app params, such as number of threads/cpu-cores.

Some of the tool's docker image info is already displayed in the listing grid, but it could be helpful to have it also available in this info dialog, especially for a user that might want to create their own app using a tool they don't have permission to edit (or for copy-paste into their own private tool for tweaking):
```javascript
tool.container.image.name
tool.container.image.tag
tool.container.image.url
tool.container.image.deprecated
tool.container.entrypoint
tool.container.uid
tool.container.working_directory
tool.version
```
These could also be added in a later PR.